bouncer/tests/broker_level_screening.ts

```diff
@@ -173,11 +221,8 @@ async function main() {
   );
   btcRefundAddress = await newAssetAddress('Btc');
 
-  await brokerLevelScreeningTestScenario(
-    '0.2',
-    true,
-    btcRefundAddress,
-    MILLI_SECS_PER_BLOCK * BLOCKS_TO_WAIT,
+  await brokerLevelScreeningTestScenario('0.2', true, btcRefundAddress, 0, (txId) =>
+    setTxRiskScore(txId, 9.0),
```

Comment:
seems strange that we always set to the wait time to 0 when we submit with monitor, but it's non-zero when not? I think it makes more sense to encode this in the closure. 
So here we just call it like this (just removing the `waitBeforeReport` parameter) and then below we can do:
```js

 async (txId) => {
      await sleep(MILLI_SECS_PER_BLOCK * 2)
      await submitTxAsTainted(txId);
    },
```

Reply:
Nice idea, done.